submissions/OpemipoBanso/HTML/kasuwa.com-registration page.html

```diff
@@ -0,0 +1,300 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+
+        <title>My Login Form</title>
+
+    </head>
+
+    <body>
+
+        <div>
+
+        <h2>Welcome to your digital food basket!</h2>
+
+        <form action="POST">
+
+                <fieldset>
+    
+                        <legend>Signup to Kasuwa</legend>
+    
+                        <label for="firstname">Firstname :</label> <br>
+                        <input type="text" name="firstname" id="firstname">
+    
+                        <br>
+    
+                        <label for="lastname">Lastname :</label> <br>
+                        <input type="text" name="lastname" id="lastname">
+    
+                        <br/>
+    
+                        <label for="email-address">Email address :</label><br>
+                        <input type="email-address" name="email-address" id="email-address" pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}">
+    
+                        <br/>
+    
+                        <label for="email-address">Password :</label> <br>
+                        <input type="password" name="password" pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}" title="Must contain at least one number and one uppercase and lowercase letter, and at least 8 or more characters">
+    
+                        <br/>
+    
+                        <label for="email-address">Confirm Password :</label> <br>
+                        <input type="password" name="password" pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}">
+    
+                        <br>
+    
+                        <label for="email-address">Email address :</label><br>
+                        <input type="email-address" name="email-address" id="email-address" pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}"> <br>
+    
+    
+                        <label for="Phone-number">Phone :</label> <br>
+                            <select>
+                                    <!-- Countries often selected by users can be moved to the top of the list. -->
+                                    <!-- Countries known to be subject to general US embargo are commented out by default. -->
+                                    <!-- The data-countryCode attribute is populated with ISO country code, and value is int'l calling code. -->
+    
+                                    <option data-countryCode="NG" value="234">Nigeria (+234)</option>
```

Comment:
Well done. Although there are better ways of loading the list of countries in a form page